packages/bpk-component-calendar/README.md

```diff
@@ -268,8 +271,40 @@ This prop determines what the colour date cell is to be displayed.
 - `CELL_TYPES.negative` - sets the calendar cell to `Panjin`
 - `CELL_TYPES.default` - sets the calendar cell to `Sky Gray Tint 02`
 
+#### `dateType` prop
+
+This property determines which selected styles will be applied to the date cell. If using ranges use `start`, `middle` and `end` to apply the correct range styles.
+
+- `DATE_TYPES.single` - sets the calendar cell to standard `Sky Blue` colour
+- `DATE_TYPES.start` - sets the calendar cell to `Sky Blue` along with the start `Sky Blue Tint 02` range background.
+- `DATE_TYPES.middle` - sets the calendar cell to `Sky Blue Tint 02` background.
+- `DATE_TYPES.end` - sets the calendar cell to `Sky Blue` along with the end `Sky Blue Tint 02` range background.
+
 ### Prop details
 
+#### calendarConfiguration
+
+An object to indicate which configuration of the calendar is being used. Choices are `single` date selection or `range` date selection.
+
+##### Single date
+
+```json
+{
+  type: 'single',
```

Comment:
Should we have a constant for `single` and `range`? `CALENDAR_SELECTION_TYPE.single`?

Reply:
Good idea!